src/tests/eolian/eolian_generation.c

```diff
@@ -65,18 +65,89 @@ _files_compare (const char *ref_filename, const char *tmp_filename)
    return result;
 }
 
+/**
+ * Returns 0 on success, -1 on failure.
+ */
+static int
+_create_response_file(const char *rsp_path, const char *eo_filename, const char *type, const char *output_filename)
+{
+    printf("Opening %s...", rsp_path);
+
+    remove(rsp_path);
+    FILE *f = fopen(rsp_path, "w");
+
+    if (!f)
+    {
+        printf("Failed!\n");
+        return -1;
+    }
+    printf("Done!\n");
+
+    printf("Writing to file...\n"
+        "    :: -S\n"
+        "    :: -I \""TESTS_SRC_DIR"\"\n"
+        "    :: -o %s:%s\n"
+        "    :: %s\n",
+        type, output_filename,
+        eo_filename
+    );
+
+    fprintf(f,
+        "-S\n"
+        "-I\n"
+        "\""TESTS_SRC_DIR"\"\n"
+        "-o\n"
+        "%s:%s\n"
+        "%s\n",
+        type, output_filename,
+        eo_filename
+    );
+
+    if (ferror(f))
+        printf("Foi no ferror");
+
+    if (fclose(f))
+        return -1;
```

Comment:
You can put those on the same line or:

```suggestion
    if (ferror(f))
      printf("Foi no ferror");

    if (fclose(f))
      return -1;
```

Reply:
I really am against same-line statements, so I'll stick with the second.

But remind me when I open for review, for now I'm just trying to understand why it crashes here (and why it reads only 1 arg and just ignores the others).

Reply:
okidoki